styles.css

```diff
@@ -111,8 +111,729 @@ nav i {
+  position: absolute;
+  left: 54px;
+  top: 185px;
+}
+
+#fifth span :nth-child(n) {
+  position: absolute;
+  left: 30px;
+  top: -447px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  border: 0;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+#fifth span :nth-child(2) {
+  left: 96px;
+}
+
+#fifth span :nth-child(3) {
+  left: 200px;
+}
+
+#fifth a button {
+  position: absolute;
+  left: -45px;
+  top: 336px;
+  width: 327px;
+}
+
+#fifth span {
+  position: absolute;
+  left: -60px;
+  top: 715px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  border: 1px solid #8993a4;
+  background: #f5f5f5;
+  color: #000;
+}
+
+#sixth {
+  position: absolute;
+  left: -16px;
+  top: 2900px;
+}
+
+#sixth h3 {
+  color: #fff;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  letter-spacing: 0.37px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-family: "Crete Round", "Times New Roman", Times, serif;
+  position: absolute;
+  left: 54px;
+  top: 100px;
+}
+
+#sixth p {
+  color: #fff;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.37px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  position: absolute;
+  left: 54px;
+  top: 185px;
+}
+
+#sixth span :nth-child(n) {
+  position: absolute;
+  left: 30px;
+  top: -447px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  border: 0;
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+#sixth span :nth-child(2) {
+  left: 96px;
+}
+
+#sixth span :nth-child(3) {
+  left: 200px;
+}
+
+#sixth a button {
+  position: absolute;
+  left: -45px;
+  top: 336px;
+  width: 327px;
+}
+
+#sixth span {
+  position: absolute;
+  left: -60px;
+  top: 715px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  border: 1px solid #8993a4;
+  background: #f5f5f5;
+  color: #000;
+}
+
+.about-me {
+  background: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  left: 500px;
+  top: 4155px;
+  position: absolute;
+  height: 1901px;
+  width: 375px;
+}
+
+.about-me h1 {
+  color: #ff6b00;
+  font-size: 72px;
+  font-weight: 400;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-family: "Crete Round", "Times New Roman", Times, serif;
+  position: absolute;
+  left: 24px;
+  top: 120px;
+}
+
+.fig2 {
+  position: absolute;
+  top: 340px;
+  right: -40px;
+}
+
+#style1 {
+  position: absolute;
+  left: 70px;
+  top: 700px;
+}
+
+#style2 {
+  position: absolute;
+  left: 70px;
+  top: 1020px;
+}
+
+#style3 {
+  position: absolute;
+  left: 70px;
+  top: 1390px;
+}
+
+.list-style {
+  list-style-type: none;
+  font-size: 22px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  line-height: 50px;
+  color: #fff;
+}
```

Comment:
- [ ] Please correct the alignment of the text to look similar to the Figma template. We could use `text-align: center;` 
- [ ] Kindly pay attention to the bullet points there is a diamond for language a square for frameworks and a circle for skills.

| **Firma template** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192123874-90639e3a-0e68-4038-a843-ffc44893bd3c.png) | ![image](https://user-images.githubusercontent.com/58642949/192123881-60a05ec1-6fbd-495b-9b83-ceee6eb08901.png) |

Reply:
@amexabee asked: The diamond bullet point is complicated to lay it out. please can I just use the circle?
- We could use images to replace bullet points and look similar to the Figam template. 👍🏽

Reply:
It is a requirement:

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).